frontend/readhistory.lua

```diff
@@ -19,7 +20,7 @@ local ReadHistory = {
 
 local function getMandatory(date_time)
     return G_reader_settings:isTrue("history_datetime_short")
-        and datetime.secondsToDate(date_time):sub(3) or datetime.secondsToDateTime(date_time)
+        and os.date(C_("Date string", "%y-%m-%d"), date_time) or datetime.secondsToDateTime(date_time)
```

Comment:
Missed that one; I would perhaps have extended `datetime.secondsToDate` to provide the century-less format directly via an extra argument instead of hard-coding it here?